yarn-project/end-to-end/src/e2e_deploy_contract/deploy_method.test.ts

```diff
@@ -35,7 +35,7 @@ describe('e2e_deploy_contract deploy method', () => {
 
   it('refused to deploy a contract instance whose contract class is not yet registered', async () => {
```

Comment:
```suggestion
  it('refused to initialize a contract instance whose contract class is not yet published', async () => {
```

Based on the new naming "initializing a contract instance" is better than "deploying a contract instance", right? If not pls don't apply this.